boards/starfive/visionfive2/doc/index.rst

```diff
@@ -0,0 +1,74 @@
+.. _visionfive2:
+
+Starfive VisionFive2 JH7110
+###########################
+
+Overview
+********
+
+Starfive VisionFive 2 is the world’s first high-performance RISC-V single board computer (SBC) with an integrated GPU. Compared with its last generation, VisionFive 2 has been fully upgraded with significant improvements in the processor work frequency, multimedia processing capabilities, scalability, etc. Its superior performance and reasonable price make VisionFive 2 the best affordable RISC-V development board ever.
+
+.. figure:: img/Visionfive2.webp
+   :align: center
+   :alt:
+
+Programming and debugging
+*************************
+
+Building
+========
+
+Applications for the ``visionfive2_jh7110`` board configuration can be built
+as usual (see :ref:`build_an_application`) using the corresponding board name:
+
+.. zephyr-app-commands::
+   :board: visionfive2
+   :goals: build
+
+To build zephyr.bin for this board use this command:
+
+.. code-block:: console
+
+   west build -p always -b visionfive2 samples/hello_world/
+
+spl_tool is a jh7110 signature tool used to generate spl header information and generate zephyr.bin.normal.out.
+`here <https://github.com/starfive-tech/Tools/tree/master/spl_tool/>`_
+
+.. code-block:: console
+
+   ./spl_tool -c -f build/zephyr/zephyr.bin
+
+This will create a new file build/zephyr/zephyr.bin.normal.out that can be flashed.
+
+Flashing
+========
+
+.. note::
+   The following steps use minicom for serial communication, feel free to use
+   any other serial terminal that supports xmodem based file transfers.
+
+git clone the vf2-loader tool from https://github.com/orangecms/vf2-loader.git and xmodem tool from https://github.com/orangecms/xmodem.rs.git side by side.
+
+#. VisionFive2 uses uart for flashing. Refer to `VisionFive2 Recovery Quick Start Guide here <https://doc-en.rvspace.org/VisionFive2/Quick_Start_Guide/VisionFive2_SDK_QSG/recovering_bootloader%20-%20vf2.html>`_
+   to connect your serial-to-usb converter. Now power on the board and using
+   minicom access board's serial and press the reset switch on the board until you see CCCCCC... prompt
+
+Copy the zephyr.bin.normal.out from build/zephyr/zephyr.bin.normal.out to previously git cloned vf2-loader/ directory and cd into it.
```

Comment:
Use double back quotes or the `:file:` role here and elsewhere when your referring to filenames. This makes them much easier to read